cmake/modules/FindZeroMQ.cmake

```diff
@@ -62,6 +62,9 @@ find_path(ZeroMQ_INCLUDE_DIR NAMES "zmq.h" "zmq_utils.h"
     DOC "ZeroMQ include directories"
     NO_DEFAULT_PATH
 )
+if(NOT ZeroMQ_INCLUDE_DIR)
```

Comment:
> The final result would, however, be as per the quite convoluted documentation (but I have tested it and can confirm), that the system one will always have priority .

This [sentence](https://github.com/Kitware/CMake/blob/666bb0e3fa0d212c03f785bd9f6bc05d29925e1f/Help/command/FIND_XXX.txt#L93) is quite misleading to not read point 6. Happened to me at least. So, in the end the cmake docs are accurate :-) @dberzano Does dberzano/FairRoot#1 work for you?